test/e2e/specs/editor/various/autosave.spec.js

```diff
@@ -0,0 +1,351 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Autosave', () => {
+	test.beforeEach( async ( { admin, page } ) => {
+		await admin.createNewPost();
+		await page.evaluate( () => window.sessionStorage.clear() );
+	} );
+
+	test.afterEach( async ( { page } ) => {
+		await page.evaluate( () => window.sessionStorage.clear() );
+	} );
+
+	test( 'should save to sessionStorage', async ( { page, pageUtils } ) => {
```

Comment:
I have removed reliance on the autosave monitor in this test. It is covered by unit tests. See: `packages/editor/src/components/autosave-monitor/test/index.js`.